helm/thingsboard/templates/ingress.yaml

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright © 2016-2020 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+{{ if .Values.ingress.enabled -}}
+{{- $fullName := include "thingsboard.fullname" . -}}
+{{- $releaseName := .Release.Name -}}
+{{- $ingressHost := .Values.ingress.host -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "thingsboard.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- with .Values.ingress.tls }}
+  tls:
+  - hosts:
+    - {{ $ingressHost }}
+    secretName: {{ $releaseName }}-tls
+  {{- end }}
+  rules:
+    - host: {{ $ingressHost }}
```

Comment:
should iterate over hosts

Reply:
Yes, related to previous modification request